pkg/cmd/pipelinerun/testdata/TestPipelineRunDescribe_cancelled_pipelinerun.golden

```diff
@@ -1,7 +1,7 @@
-Name:           pipeline-run
-Namespace:      ns
-Pipeline Ref:   pipeline
-Timeout:        1h0m0s
+Name:                  pipeline-run
+Namespace:             ns
+Pipeline Ref:          pipeline
+Timeout(Deprecated):   1h0m0s
```

Comment:
should we show `Deprecated` with the timeout or revert this back ?
/cc @vdemeester @piyush-garg @pradeepitm12